css/style.css

```diff
@@ -1,3 +1,197 @@
-h1 {
-  color: purple;
+body {
+	text-align: center;
```

Comment:
For some reason my CSS line indents don't show on github when I commit, looking into if this is just me/something I need to change.